src/development/environment-variables.md

```diff
@@ -3,40 +3,61 @@
-you can list the environment variables by running:
+## Adding environment variables
+
+New custom environment variables can be set [through the web interface](administration/web/configure-environment.html#settings), or using the CLI. For example, this command adds the variable `foo` to the `$PLATFORM_VARIABLES` object...
+
+```bash
+$ platform variable:set foo bar
+```
+## Inspecting and listing environment variables
+
+Custom environment variables which you have defined can be viewed [via the Web Interface](administration/web/configure-environment.html#settings) or using the CLI...
+
+```bash
+$ platform variables
+
++---------+-------+-----------+------+
+| ID      | Value | Inherited | JSON |
++---------+-------+-----------+------+
+| env:FOO | bar   | No        | No   |
++---------+-------+-----------+------+
+```
+You can also list all the environment variables when logged in via SSH to an environment (with the cli: `platform ssh`):
 
 ```bash
 web@myp3bmdujgzqe-master--php:~$ export
 ```
-Since they are **base64-encoded JSON object** that maps variable names
-to variable values, you can decode the value of user defined variables by
+Some environment variables are **base64-encoded JSON objects** that map variable names
+to variable values. You can decode the value of user defined variables by
 running:
 
 ```bash
 echo $PLATFORM_VARIABLES | base64 --decode
 {"myvar": "this is a value"}
 ```
 
-In this example we defined for one of our environments a variable we called
+In this example we defined a variable for one of our environments that we called
 "myvar".
 
-> **note**
-> Variables are **hierarchical**, so if a variable is not overriden in an environment, it will take the value it has in the parent environment and use it as `inherited`.
-> This allows you to define your development variables only once, and use them on all the child environments.
+## Environment inheritance
+
+Variables are **hierarchical**, so if a variable is not overridden in an environment, it will take the value it has in the parent environment and use it as `inherited`.
+
+This allows you to define your development variables only once, and use them on all the child environments.
 
 > **note**
 > If a variable is added in the parent environment after the child environments are created, the child environments have to be re-deployed in order to load the inherited variables.
 
 ## Platform.sh variables
 
-Environment variables that are specific to Platform.sh are exposed in
-the runtime (*ie. PHP*) and prefixed with `PLATFORM_*`.
+Environment variables that are specific to Platform.sh are exposed in the runtime (*ie. PHP*) and prefixed with `PLATFORM_*`. These variables are used to provide a a range of information about your environment and services for use in application configuration. For example, for Drupal, we use the **PLATFORM_RELATIONSHIPS** variable to configure your `settings.local.php`.
```

Comment:
The ie here should be an `e.g.`, since it's an example, not an "in other words". :smile: